checkoutmanager/dirinfo.py

```diff
@@ -156,6 +185,24 @@ class BzrDirInfo(DirInfo):
     vcs = 'bzr'
 
     @capture_stdout
+    def cmd_rev(self):
+        print(self.directory)
+        os.chdir(self.directory)
+        print(system("bzr log -r-1"))
+
+    @capture_stdout
+    def cmd_cu(self):
+        output = system("bzr missing")
+        output = output.strip()
+        output_lines = output.split('\n')
+        if "Branches are up to date." in output_lines:
```

Comment:
Lines like this are very brittle. I'm speaking from experience with zest.releaser here. Problems I've seen:

- Suddenly the initial letter 'B' becomes a 'b' in a newer version of bzr (or one of the other ones).

- Someone has a localized shell and bzr nicely renders the message in french/german/dutch/whatever...

I'm not against testing for the string output like you're doing: there's not much else we can do. I've tried looking at exit codes in zest.releaser and that was even worse :-) There are two things we can do:

- At least do a `.lower()` before comparing.

- Add a short note in the readme for the "cu" command that it is inherently a bit fragile as it depends on the english textual output of the various commands.